src/walletSdk/Anchor/Sep24.ts

```diff
@@ -46,13 +46,15 @@ export class Sep24 {
     authToken,
     lang,
     extraFields,
+    destinationMemo,
     destinationAccount,
   }: Sep24PostParams): Promise<Sep24PostResponse> {
     return this.flow({
       assetCode,
       authToken,
       lang,
       extraFields,
+      destinationMemo,
```

Comment:
Nice addition. Should we also add memo to `withdraw()` ? According to the SEP it should be possible: https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#withdraw

Reply:
Ya i noticed that too but the Kotlin sdk didn't have it so I left it off

@Ifropc any reason for that? or can I go ahead and memos for withdrawals too

Reply:
It's deprecated and should be derived from SEP-10 token instead. I don't know of any anchor that doesn't support this functionality

Reply:
Makes sense, I've only seen cases for deposit memo